bundles/binding/org.openhab.binding.velux/README.md

```diff
@@ -203,34 +174,31 @@ rule "PushButton of group gV"
 
 ### Debugging
 
-For those who are interested in more detailed insight view of the processing of this binding, a deeper look can be achieved by increased loglevel.
-
-Try to add the following lines to  `/etc/openhab/logback.xml` for common standalone installation:
+For those who are interested in more detailed insight of the processing of this binding, a deeper look can be achieved by increased loglevel.
 
+During startup of normal operations, there should be only some few messages within the logfile, like:
 ```
-    <!-- Velux binding logging -->
-    <logger name="org.openhab.binding.velux" level="DEBUG">
-        <appender-ref ref="TMPEVENTFILE" />
-    </logger>
+[INFO ] [.b.velux.internal.VeluxBinding] - Velux refresh interval is 3600000 seconds.
+[INFO ] [.service.AbstractActiveService] - velux Refresh Service has been started
+[INFO ] [.b.velux.internal.VeluxBinding] - veluxConfig[bridgeIPAddress=192.168.41.1,bridgeTCPPort=80,bridgePassword=********,timeoutMsecs=2000,retries=6]
 ```
 
+## Supported/Tested Firmware Revisions
 
-During startup of normal operations, there should be only some few messages within the logfile, like:
+The Velux Bridge in API version One (firmware version 0.1.1.*) allows to activate a set of predefined actions, so called scenes. Therefore besi
+de the bridge, only one main thing exists, the scene element. Unfortunatelly even the current firmware version 0.1.1.0.44.0 does not include en
+hancements on this fact.
 
-```
-2018-07-23 21:27:09.434 [INFO ] [.b.velux.internal.VeluxBinding] - Velux refresh interval is 3600000 seconds.
-2018-07-23 21:27:09.435 [INFO ] [.service.AbstractActiveService] - velux Refresh Service has been started
-2018-07-23 21:27:09.435 [INFO ] [.b.velux.internal.VeluxBinding] - veluxConfig[bridgeIPAddress=192.168.41.1,bridgeTCPPort=80,bridgePassword=********,timeoutMsecs=2000,retries=6]
-
-```
+| Firmware revision | Release date | Description                                                             |
+|:-----------------:|:------------:|-------------------------------------------------------------------------|
+| 0.1.1.0.41.0      | 2016-06-01   | Default factory shipping revision.                                      |
+| 0.1.1.0.42.0      | 2017-07-01   | N/A                                                                     |
+| 0.1.1.0.44.0      | 2017-12-14   | N/A                                                                     |
 
-## Famous last words
+## Unknown Velux devices
 
-All known <B>Velux<B> devices can be handled by this binding. In fact, there might be some new one which will be reported within the logfiles. Therefore, if you recognize an error message like:
+All known <B>Velux<B> devices can be handled by this binding. However, there might be some new ones which will be reported within the logfiles.Therefore, error messages like the one below should be reported to the maintainers so that the new Velux device type can be incorporated."
```

Comment:
You have two opening `<B>` elements here.  The second one should be a closing `</B>`.